spec/tests/osm_construction_set_spec.rb

```diff
@@ -51,6 +53,7 @@
     expect(honeybee).not_to be nil
     expect(honeybee[0][:type]).to eq 'ConstructionSetAbridged'
     expect(honeybee[0][:identifier]).to eq '189.1-2009 - CZ1 - Office'
+    #expect(honeybee[0][:display_name]).to eq 'Test Name áéíóúüñ¿¡'
```

Comment:
@chriswmackey I tried to use the to_json instead of the pretty json method but I am still getting an error

     Failure/Error: expect(honeybee[0][:display_name]).to eq 'Test Name ├í├⌐├¡├│├║├╝├▒┬┐┬í'

       expected: "Test Name \u00E1\u00E9\u00ED\u00F3\u00FA\u00FC\u00F1\u00BF\u00A1"
            got: "Test Name \xC3\xA1\xC3\xA9\xC3\xAD\xC3\xB3\xC3\xBA\xC3\xBC\xC3\xB1\xC2\xBF\xC2\xA1"

Reply:
Hey, @tanushree04 ,

Sorry for the late response. If you aren't getting the same error that I did, at least we can say that the JSON is able to be written when using the `to_json` method.

It looks like the `to_json` method is escaping the characters as hex codes instead of unicodes. [See here for a reference](https://stackoverflow.com/questions/33415388/whats-the-difference-between-hex-code-x-and-unicode-u-chars).

I just checked and the Python JSON modules that we are using really need the characters to be escaped as unicode (using the hex code results in an error).

@macumber , could you recommend a way to ensure that the characters are escaped as unicode characters and not hex characters?

Reply:
@chriswmackey (construction_set.displayName.get).force_encoding("UTF-8") seems to work, I will push the changes.